api/build.gradle

```diff
@@ -403,7 +403,7 @@ dependencies {
     compile "io.swagger:swagger-annotations:1.5.16"
     compile "javax.inject:javax.inject:1"
     compile "org.liquibase:liquibase-core:3.10.3"
-    compile "mysql:mysql-connector-java:8.0.28"
+    compile "mysql:mysql-connector-java:8.0.22"
```

Comment:
if we want 8.0.28 back, we may need to upgrade liquibase to 4.3.x

Reply:
FYI https://security.snyk.io/vuln/SNYK-JAVA-MYSQL-1766958